### PR TITLE
First version

### PR DESCRIPTION
**Pull request by men32z**

### CODE REVIEW

File: README.md
Comment (bolah2009) [inline README.md:1]: #### `AWESOME`

Nice job adding a descriptive readme... :star2:

File: README.md
Comment (bolah2009) [inline README.md]: #### `OPTIONAL`

How about we do this like this :thinking: ...

> **You can find the original project specifications [here](https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms)**
> Which can be achieved by doing this :point_down: 
> **`You can find the original project specifications [here](https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms)`**

This makes it look more professional. :wink: 

###### Find more details on using markdown [here](https://guides.github.com/features/mastering-markdown/).

File: README.md
Comment (bolah2009) [inline README.md:11]: #### `OPTIONAL`

Check this comment on using markdown links.

:bulb: You could also add a [live link](https://raw.githack.com/men32z/sing-up-form/first-version/index.html)

###### Note that if you delete `first-version` branch after merging the above link won't work. You can use `Github Pages` or change to this link: `https://raw.githack.com/men32z/sing-up-form/first-version/index.html`.

###### Find more details on using **Github Pages** [here](https://guides.github.com/features/mastering-markdown/).

File: README.md
Comment (bolah2009) [inline README.md]: #### `OPTIONAL`

Let's do this too instead. :wink: `Clone Screenshot`. See below for the result.

Clone Screenshot

###### Find more details on using markdown [here](https://guides.github.com/features/mastering-markdown/).

File: css/styles.css
Comment (bolah2009) [inline css/styles.css]: #### `HINT`

Most of `stickler-ci` checks are `(indentation) [error]`, to make this easier. Format the code. :shower:

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

Screenshot of how to format code using VS Code

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

Consider adding `lang="en"` (or variant) to the `html` start tag, this is good for internationalisation. Also, check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics)

Take advantage of [HTML CHECKER](https://validator.w3.org) by checking your code with this online tool. It spots out common mistakes/omission in HTML codes. :+1:

File: index.html
Comment (bolah2009) [inline index.html:7]: #### `HINT`

It seems we are missing the `<title>` element. :bulb:

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

:bug: Let's use `type="tel"` instead, as `type="phone"` is not a valid `input` `type` value.

:bulb: Using `type="tel"` for Phone number input makes adding custom validation and handling of phone numbers more convenient among other dedicated features.

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel).

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

Please, always use `alt` attribute for `` element. :bulb: 

This is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

File: index.html
Comment (bolah2009) [inline index.html:28]: #### `AWESOME` 

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. :+1:

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

:bulb: Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```
- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

File: index.html
Comment (bolah2009) [inline index.html]: :bulb: See [the comment](https://github.com/men32z/sing-up-form/pull/1#discussion_r315018543) on associating `input` element with `label` element.

File: index.html
Comment (bolah2009) [inline index.html]: :bulb: See [the comment](https://github.com/men32z/sing-up-form/pull/1#discussion_r315018543) on associating `input` element with `label` element.

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml:3]: #### `HINT`

:bulb: To avoid making multiple commits while fixing linter errors, try this. :point_down: 

>  :boom: 
> - run [stylelint](https://stylelint.io/) in your local env:
>     - run `npm install stylelint stylelint-config-recommended --save-dev`  (not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
>     - copy [stylelint.config.js](./stylelint.config.js) to the root directory of your project
>     - run `npx stylelint .`
>     - fix linter errors

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job Team! :muscle: :sunglasses: 

The clone looks good to me and meets *most* requirements, a few changes before :+1:, check the comments below.

- Fix linter errors/warnings. [HINT](https://github.com/men32z/sing-up-form/pull/1#discussion_r315021708)
- Check HTML code with [HTML Validator](https://validator.w3.org/), currently there are some [error/warnings](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fmen32z%2Fsing-up-form%2Ffirst-version%2Findex.html) need to be fixed.
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.

Keep up the good work and best of luck on the next submission. :+1:

Happy Coding!  :dizzy: and Have a Nice Day. :smile:

Cheers
[Bola](https://bolabuari.com/)

###### Note that comments tagged `OPTIONAL` are strongly recommended but not required to approve the project.

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — APPROVED** [review]
### Awesome team :clap: 
#### Your form is very beautiful. Great work making the suggested changes :+1: 
I'm approving this project :smile: 
Go on and merge to master. Good luck with the next one. 

[Next Project] 
Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. embedding-images-and-video, html-forms, responsive-design, etc) and do not merge any feature branch to master until it's been approved.

**Pull request merged**
